aasemble/django/apps/api/tests.py

```diff
@@ -733,10 +733,19 @@ def test_get_correct_mirror_for_user(self):
 
     @mock.patch('aasemble.django.apps.mirrorsvc.tasks.refresh_mirror')
     def test_refresh_mirror_status(self, refresh_mirror):
-        mirror = self.test_create_mirror()
-        response = self.client.post(mirror['self'] + 'refresh/')
+        mirror_dict = self.test_create_mirror()
+        mirror_id = int(mirror_dict['self'].split('/')[-2])
+        refresh_mirror.assert_called_with(mirror_id)
```

Comment:
@sorenh I'm doing something wrong with the refresh_mirror.assert_called_with(mirror_id) statement. The refresh_mirror doesn't seem to be getting called. Need help with understanding where the bug is.